app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,50 @@
+class AddressesController < ApplicationController
+  before_action :set_user, :set_address
```

Comment:
Do we want to run these filters on all of the actions?

Reply:
No